language/tools/vm-genesis/src/lib.rs

```diff
@@ -556,18 +575,22 @@ pub fn generate_genesis_type_mapping() -> BTreeMap<Vec<u8>, StructTag> {
 
 /// Generates an artificial set of OperatorAssignments using the given node configurations for
 /// testing.
-pub fn operator_assignments(node_configs: &[NodeConfig]) -> Vec<OperatorRegistration> {
```

Comment:
Aah.. annoying that OperatorRegistration and OperatorAssignment had the same types.. otherwise the compiler would have told us that this was wrong 😥